### ISSUE TITLE
DuoFern Hutschienen Schaltaktor 2-Kanal

### ISSUE THREAD

**Issue by BitSJ9** [bombadiltom/homebridge-rademacher-homepilot#128]
Hallo,
ich wollte kurz abklären ob der neue Rademacher DuoFern Hutschienen Schaltaktor 2-Kanal Typ 9470-4 auch schon hinzugefügt ist?

MfG,

Stefan

**Comment by bombadiltom** [comment]
nein, siehe https://github.com/bombadiltom/homebridge-rademacher-homepilot/blob/master/index.js

Da fehlt noch 35200262, denke ich

**Comment by bombadiltom** [comment]
und 35200462 für den Dimmer

**Comment by BitSJ9** [comment]
Ich wollte diese jetzt verbauen, kann ich aber damit rechnen das diese irgendwann funktionieren, oder sollte ich besser die normalen Unterputz Aktoren verbauen?MfG Stefan Von Stefan's iPhone gesendetAm 04.09.2023 um 14:10 schrieb Tom Lorenz ***@***.***>:﻿
Closed #128 as completed.

**Comment by bombadiltom** [comment]
die sollten mit dem release 0.9.6 funktionieren, da ich erwarte, dass sie sich wie die Unterputz Aktoren verhalten, garantieren kann ich das aber leider nicht

**Comment by BitSJ9** [comment]
Ich habe den Aktor gerade getestet funktioniert. 🔝👍🏼Von Stefan's iPhone gesendetAm 04.09.2023 um 15:46 schrieb Tom Lorenz ***@***.***>:﻿
die sollten mit dem release 0.9.6 funktionieren, da ich erwarte, dass sie sich wie die Unterputz Aktoren verhalten, garantieren kann ich das aber leider nicht
